tests/test_bundle.py

```diff
@@ -105,7 +105,7 @@ def test_bundle_paging(self):
             ))
             expected_files = manifest['files']
 
-        for replica in Replica:
+        for replica in (Replica.aws, Replica.gcp):
```

Comment:
was this changed to allow the gcp replica to be tested as well?

Reply:
Yep!